SDWebImage/SDImageCachesManager.m

```diff
@@ -103,7 +119,9 @@ - (void)storeImage:(UIImage *)image imageData:(NSData *)imageData forKey:(NSStri
     if (!key) {
         return;
     }
-    NSArray<id<SDImageCache>> *caches = [self.caches copy];
+    SD_LOCK(self.cachesLock);
+    NSArray<id<SDImageCache>> *caches = self.caches;
```

Comment:
@dreampiggy @kinarob This is not thread-safe. 🤔

Reply:
What means, not thread-safe ?

It's rare that people modify this value. It's designed to be a setup logic in `AppDelegate.m` actually.

Previous version, see: SDImageLoadersManager, using a mutable array, each method call need to `copy` a instance. This one does not.

Reply:
This code actually already in master branch from v4.4.0.

See #2274